mslib/msui/qt5/ui_performance_dockwidget.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'mslib/msui/ui/ui_performance_dockwidget.ui'
+#
+# Created by: PyQt5 UI code generator 5.9.2
+#
+# WARNING! All changes made in this file will be lost!
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+class Ui_PerformanceDockWidget(object):
+    def setupUi(self, PerformanceDockWidget):
+        PerformanceDockWidget.setObjectName("PerformanceDockWidget")
+        PerformanceDockWidget.resize(767, 123)
+        self.gridLayout = QtWidgets.QGridLayout(PerformanceDockWidget)
+        self.gridLayout.setContentsMargins(9, 9, 9, 9)
+        self.gridLayout.setVerticalSpacing(1)
+        self.gridLayout.setObjectName("gridLayout")
+        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
+        self.horizontalLayout_2.setContentsMargins(-1, 8, -1, -1)
+        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
+        self.label_2 = QtWidgets.QLabel(PerformanceDockWidget)
+        self.label_2.setIndent(5)
+        self.label_2.setObjectName("label_2")
+        self.horizontalLayout_2.addWidget(self.label_2)
+        self.lbAircraftName = QtWidgets.QLabel(PerformanceDockWidget)
+        self.lbAircraftName.setIndent(3)
+        self.lbAircraftName.setObjectName("lbAircraftName")
+        self.horizontalLayout_2.addWidget(self.lbAircraftName, 0, QtCore.Qt.AlignHCenter)
+        self.pbLoadPerformance = QtWidgets.QPushButton(PerformanceDockWidget)
+        self.pbLoadPerformance.setObjectName("pbLoadPerformance")
+        self.horizontalLayout_2.addWidget(self.pbLoadPerformance, 0, QtCore.Qt.AlignRight)
+        self.gridLayout.addLayout(self.horizontalLayout_2, 1, 1, 1, 1)
+        self.horizontalLayout_4 = QtWidgets.QHBoxLayout()
+        self.horizontalLayout_4.setContentsMargins(-1, -1, 0, -1)
+        self.horizontalLayout_4.setObjectName("horizontalLayout_4")
+        self.label_4 = QtWidgets.QLabel(PerformanceDockWidget)
+        self.label_4.setIndent(1)
+        self.label_4.setObjectName("label_4")
+        self.horizontalLayout_4.addWidget(self.label_4)
+        self.dsbFuel = QtWidgets.QDoubleSpinBox(PerformanceDockWidget)
```

Comment:
Modifying this spin box does not update the Table.

Reply:
Fixed this problem in the latest commit.